brian2/devices/device.py

```diff
@@ -412,10 +414,10 @@ def spike_queue(self, source_start, source_end):
         # Use the C++ version of the SpikeQueue when available
         try:
             from brian2.synapses.cythonspikequeue import SpikeQueue
-            logger.info('Using the C++ SpikeQueue', once=True)
+            logger.diagnostic('Using the C++ SpikeQueue', once=True)
         except ImportError:
             from brian2.synapses.spikequeue import SpikeQueue
-            logger.info('Using the Python SpikeQueue', once=True)
+            logger.diagnostic('Using the Python SpikeQueue', once=True)
```

Comment:
OK to making this a post-2.0 issue (I'll open it now) and also to having Python spike queue being info level and not warning level.